src/components/InformationZone/InformationZone.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+class InformationZone extends React.Component {
```

Comment:
is there a reason to use class style instead func comps?

Reply:
I used the `ErrorBoundary` file as reference.
But I prefer the func components as well.
Going to change!